packages/process/src/node/raw-process.ts

```diff
@@ -18,11 +18,13 @@ import { injectable, inject, named } from 'inversify';
 import { ProcessManager } from './process-manager';
 import { ILogger } from '@theia/core/lib/common';
 import { Process, ProcessType, ProcessOptions } from './process';
-import { ChildProcess, spawn } from 'child_process';
+import { ChildProcess, spawn, fork } from 'child_process';
 import * as stream from 'stream';
 
 export const RawProcessOptions = Symbol('RawProcessOptions');
 export interface RawProcessOptions extends ProcessOptions {
+    /** default: spawn */
+    type?: 'spawn' | 'fork';
```

Comment:
Could you document what each option does?  To somebody who isn't very familiar with node's `child_process`, it's not super clear.

Reply:
Inline using jsdoc markup?

Reply:
The comment won't appear in the .d.ts unless it is actually jsdoc syntax (starting with `/**`).  The idea of it appearing ni the .d.ts is that somebody consuming the module from npm will be able to

1. see the documentation, since if they ctrl-click the field, it will lead them most likely to the .d.ts
2. get the documentation by hovering:

![screenshot_2018-11-26_11-35-26](https://user-images.githubusercontent.com/1758287/49028293-0ce89380-f170-11e8-9bba-6b288ed895fe.png)

I formatted and tweaked your comment a little bit, here's what I suggest:

```
    /**
     * Whether to spawn a new process (`spawn`) or execute a Javascript module
     * using the current Node interpeter (`fork`).
     *
     * If undefined, the default is `spawn`.
     *
     * For more information please refer to the spawn and fork functions
     * of Node's child_process module:
     *
     *   https://nodejs.org/dist/latest-v10.x/docs/api/child_process.html
     */
```

Reply:
Sure, I just copied the format already used in the file

Reply:
boom!